wolfssl/wolfcrypt/asn.h

```diff
@@ -2138,11 +2138,11 @@ WOLFSSL_LOCAL int EncodePolicyOID(byte *out, word32 *outSz,
 WOLFSSL_API int CheckCertSignature(const byte*,word32,void*,void* cm);
 WOLFSSL_LOCAL int CheckCertSignaturePubKey(const byte* cert, word32 certSz,
         void* heap, const byte* pubKey, word32 pubKeySz, int pubKeyOID);
-#ifdef OPENSSL_EXTRA
+#if defined(OPENSSL_EXTRA) || defined(WOLFSSL_SMALL_CERT_VERIFY)
```

Comment:
build.../tmp/workspace/PRB-multi-test-script/wolfssl/./wolfssl/wolfcrypt/asn.h:2138:47: warning: all parameters should be named in a function [readability-named-parameter]
WOLFSSL_API int CheckCertSignature(const byte*,word32,void*,void* cm);
^
/*cert*/  /*certSz*/  /*heap*/
make[2]: *** [Makefile:7321: